pkg/agent/kube.go

```diff
@@ -117,6 +118,22 @@ func getOpenServiceCatalogAnnotations(svc v1.Service) *api.Osca {
 	}
 }
 
+func fetchCertificate(kubeEnv *KubeEnv) []byte {
+	service, err := kubeEnv.Client.CoreV1().Services("infrastructure").Get(context.Background(), "sealed-secrets-controller", metav1.GetOptions{})
+	if err != nil {
+		logrus.Errorf("could not get sealed secret service: %s", err)
```

Comment:
Make it debug level. It can be business as usual to not have sealed secrets enabled